docs/admin/authorization/rbac.md

```diff
@@ -186,7 +186,8 @@ rules:
   verbs: ["update", "get"]
 ```
 
-Notably, `resourceNames` can NOT be used to limit requests using the "create" verb because
+Notably, if `resourceNames` are set, then the verb must not be list, watch, create, or deletecollection.
+`resourceNames` can NOT be used to limit requests using the "create" verb because
```

Comment:
would something like this be clearer:
```
Because resource names are not present in the URL for create, list, watch, and deletecollection API requests,
those verbs would not be allowed by a rule with resourceNames set, since the resourceNames portion of the
rule would not match the request.
```